snyk_threadfix/main.py

```diff
@@ -252,7 +253,8 @@ def main(args):
     if not token_is_valid:
         raise SnykTokenInvalidError('invalid token')
 
-    client = snyk.SnykClient(snyk_token)
+    user_agent_string = 'snyk-threadfix/%s' % __version__
```

Comment:
I'd definitely recommend starting to use `black` and `isort`, consistency of formatting is super useful when it comes to maintenance over time.

Reply:
Yes! I will. I have it on the board as BDE-16: https://snyksec.atlassian.net/browse/BDE-16